zuul.d/jobs.yaml

```
@@ -22,6 +22,9 @@
       - LICENSE
       - OWNERS
       - .*/*.md
+    vars:
+      cifmw_set_openstack_containers_overrides:
+        RELATED_IMAGE_OS_CONTAINER_IMAGE_URL_DEFAULT: "quay.io/steveb/edpm-hardened-uefi:18.0-steveb-20231204"
```

```suggestion
      cifmw_edpm_deploy_baremetal_update_os_containers: true
      cifmw_set_openstack_containers_overrides:
        RELATED_IMAGE_OS_CONTAINER_IMAGE_URL_DEFAULT: "quay.io/steveb/edpm-hardened-uefi:18.0-steveb-20231204"
```
@steveb As per this https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml#L41 we need to set `cifmw_edpm_deploy_baremetal_update_os_containers: true` to run set_openstack_containers role.